gqcpy/src/QCMethod/HF/UHF/UHFSCFEnvironment_bindings.cpp

```diff
@@ -88,6 +88,19 @@ void bindUHFSCFEnvironment(py::module& module) {
             "coefficient_matrices_beta",
             &GQCP::UHFSCFEnvironment<double>::coefficient_matrices_beta)
 
+        .def_readonly(
+            "spin_resolved_density_matrices",
```

Comment:
Is there possibly a reason why the Python method isn't called `density_matrices`?

Reply:
I thought this would convey more clearly what the method returned, since we also have bindings for the separate alpha and beta matrices. But I can rename it to just `density_matrices` if you'd prefer that.